app/src/androidTest/java/com/sdp/movemeet/view/home/LoginActivityTest.java

```diff
@@ -24,12 +24,11 @@
 
 public class LoginActivityTest {
 
-    public static final String mEmail = "email";
```

Comment:
good rename !

Reply:
Yep, this is again to follow our variable naming convention agreed two weeks ago 👍.